modules/cudn-status-conditions.adoc

```diff
@@ -80,3 +80,123 @@ h|Message
 |`Network allocation failed for at least one node: [<node_name>], check UDN events for more info.`
 
 |===
+
+.Invalid `mtu` scenarios types for the `ClusterUserDefinedNetwork` CR
+[cols="2a,2a,3a,3a",options="header"]
+|===
+
+|Condition type
+3+|Reason, Message, Resolution
+
+.6+|`invalid mtu`
+3+|One of the following messages is returned when the `mtu` is set incorrect:
+h|Reason
+h|Message
+h|Resolution
+
+|The `mtu` is set higher than `65536`.
+|`spec.network.localnet.mtu` in body should be less than or equal to `65536`.
+|You must the `mtu` lower than `165536`.
+
+|The `mtu` must bet set lower than `5761`.
+|`spec.network.localnet.mtu` in body should be greater than or equal to `576`.
+|You must set `mtu` higher than `576`
+
+|The `mtu` must be at least `1280` when IPv6 subnet is set.
+|`MTU should be greater than or equal to 1280 when an IPv6 subnet is used`
+|You must set the `mtu` higher than `1280` when you have the subnet set on a cluster using IPv6.
```

Comment:
or equal, right ? 

And the last premise is confusing (to me). I'd say something like: "... when you have an IPv6 subnet defined on the UDN configuration"